spec/lib/embed/media_tag_spec.rb

```diff
@@ -39,8 +39,17 @@
       expect(auth_url).to match(%r{https?://stacks\.stanford\.edu.*/auth_check})
     end
 
-    it 'includes a height attribute equal to the body height minus some px to make way for the thumb slider' do
-      expect(subject).to have_css('video[height="276px"]')
+    context 'single video' do
+      let(:purl) { single_video_purl }
+      it 'includes a height attribute equal to the body height' do
+        expect(subject).to have_css("video[height='#{viewer.body_height}px']")
+      end
+    end
+
+    context 'multiple videos' do
+      it 'includes a height attribute equal to the body height minus some px to make way for the thumb slider' do
+        expect(subject).to have_css('video[height="276px"]')
```

Comment:
That's kind of indicated in the test above.  `viewer.body_height` is set above and there is a test indicating that in the context of a single video purl, the height of the video should equal the viewer body height.

In this case, I prefer to not include implementation details in the comments since they are less likely to change than the test itself.